app/Http/Transformers/EventTransformer.php

```diff
@@ -16,9 +16,17 @@ class EventTransformer extends TransformerAbstract
     public function transform(Event $event)
     {
         return [
-            'event_id' => (string) $event->id,
+            'event_id' => $event->id,
```

Comment:
ah I actually didn't notice that it was a string there at first! But it's an `INT` in the database, is it better to reflect what it is in the database?